src/main/java/org/spdx/sbom/gradle/SpdxSbomPlugin.java

```diff
@@ -111,15 +102,6 @@ private void createTaskForTarget(
 
                   List<String> configurationNames = target.getConfigurations().get();
                   for (var configurationName : configurationNames) {
-                    Provider<Set<ResolvedArtifactResult>> artifacts =
-                        project
-                            .getConfigurations()
-                            .getByName(configurationName)
-                            .getIncoming()
-                            .getArtifacts()
-                            .getResolvedArtifacts();
-                    t.getResolvedArtifacts().putAll(artifacts.map(new ArtifactTransformer()));
-
```

Comment:
This was causing resolution issues in my project because of variants and mismatched attributes.

Reply:
can you describe the issue a little more for us to understand?